daphne/src/vdaf/prio3.rs

```diff
@@ -147,6 +153,34 @@ pub(crate) fn prio3_prepare_init(
                 VdafMessage::Prio3ShareField128(share),
             ))
         }
+        Prio3Config::SumVec { bits, len } => {
+            let vdaf: Prio3<
+                prio::flp::types::SumVec<
+                    prio::field::Field128,
+                    prio::flp::gadgets::ParallelSum<
+                        prio::field::Field128,
+                        prio::flp::gadgets::BlindPolyEval<prio::field::Field128>,
+                    >,
+                >,
+                prio::vdaf::prg::PrgSha3,
+                16,
+            > = Prio3::new_sum_vec(2, *bits, *len)?;
+            // TODO(tholop): we don't need vec of Prio3Field128, right?
+            // Since the Prio3PrepareState<F, SEED_SIZE> will contain a Vec<F>
+            // https://github.com/divviup/libprio-rs/blob/3615c6d92dea9c5d4ab8faa9700970f685729fe1/src/vdaf/prio3.rs#L908
```

Comment:
I believe we already have a variant of `VdafState` that we'll need: https://github.com/cloudflare/daphne/blob/main/daphne/src/vdaf/mod.rs#L80

Note that libprio's `Prio3PrepareState` is not specific to a `Type`.